src/info/channel.go.tmpl

```diff
@@ -11,21 +11,29 @@
 	{{ $channel = $args.Get 0 }}
 {{ end }}
 
-{{ $types := cslice "Text" "DM" "Voice" "Group DM" "Category" "News" "Store" }}
+{{ $types := sdict 
+	"0" "Text" "1" "DM" "2" "Voice" "3" "Group DM" "4" "Category" "5" "Announcement" 
+	"10" "Thread - Announcement" "11" "Thread - Public" "12" "Thread - Private" 
+	"13" "Stage" "14" "Directory" "15" "Forum" "16" "Media"
+}}
+{{ $type := $types.Get (json $channel.Type) }}
 {{ $nsfw := "No" }}
 {{ $parent := "*None set*" }}
 {{ if $channel.NSFW }} {{ $nsfw = "Yes" }} {{ end }}
-{{ with $channel.ParentID }} {{ $parent = printf "<#%d>" . }} {{ end }}
-{{ $createdAt := div $channel.ID 4194304 | add 1420070400000 | mult 1000000 | toDuration | (newDate 1970 1 1 0 0 0).Add }}
-{{ sendMessage nil (cembed
+{{ with $channel.ParentID }}
+	{{ $parent = printf "<#%d>" . }}
+	{{ if (getChannel .).IsForum }} {{ $type = "Thread - Forum" }} {{ end}}
+{{ end }}
+{{ $createdAt := snowflakeToTime $channel.ID }}
+{{ sendMessage nil ($e:=cembed
 	"title" (printf "❯ Info for #%s" $channel.Name)
 	"fields" (cslice
 		(sdict "name" "❯ ID" "value" (str $channel.ID) "inline" true)
 		(sdict "name" "❯ Topic" "value" (or $channel.Topic "*None set*") "inline" true)
 		(sdict "name" "❯ Parent Channel" "value" $parent "inline" true)
 		(sdict "name" "❯ NSFW" "value" $nsfw "inline" true)
-		(sdict "name" "❯ Position" "value" (str (add $channel.Position 1)) "inline" true)
-		(sdict "name" "❯ Type" "value" (index $types $channel.Type) "inline" true)
+		(sdict "name" "❯ Position" "value" (add $channel.Position 1 | str) "inline" true)
```

Comment:
Please revert this. There is no benefit when there are only two function calls.

Reply:
Was done for readability, but has been reverted.